opengemini/client.go

```diff
@@ -23,6 +23,11 @@ type Client interface {
 	WritePoint(ctx context.Context, database string, point *Point, callbackFunc WriteCallback) error
 	// WriteBatchPoints write batch points to assigned database
 	WriteBatchPoints(database string, bp []*Point) error
+	// WritePointWithRp write single point into special retention policy and assigned database
+	WritePointWithRp(ctx context.Context, database string, rp string, point *Point, callbackFunc WriteCallback) error
```

Comment:
can we keep order consitent with design doc?

Reply:
done